src/test/acceptance/partner_profile.js

```diff
@@ -63,5 +63,12 @@ describe("Partner profile page", () => {
       $.html().should.containEql("Contact")
       $.html().should.containEql("New York")
     })
+
+    it("should not render a page for /system/up healthcheck endpoint", async () => {
```

Comment:
@damassi and @alloy notably this test doesn't fail if we revert the changes to setup.js :( (it will fail if the system/* routes aren't present though).

@alloy: Is there a good way to write a mock expectation that in acceptance tests that a service isn't called (gravity in this case)? Chris and I weren't aware of a pattern here for Nightmare-based tests like this.

Reply:
I've never used this but maybe [nock](https://github.com/nock/nock#expectations) is relevant? We already [included](https://github.com/artsy/force/blob/9f903d3541a1e604b6d1b1f0f3b6c58eb0712dbb/package.json#L166) nock for other purposes. Also, what if we close Gravity server for this test. Maybe it will fail for previous implementation?

Reply:
@starsirius Thanks for the pointer, will give myself a couple of hours to see if I can figure out a nock-based approached here!

Reply:
Since this is hitting grav every 5 seconds should we go ahead and merge and then follow up with an updated test?

Reply:
Alas, I have no experience with this test setup yet. Glad y'all figured it out!